shell/platform/windows/flutter_windows_view.h

```diff
@@ -221,10 +224,7 @@ class FlutterWindowsView : public WindowBindingHandlerDelegate,
   virtual std::unique_ptr<TextInputPlugin> CreateTextInputPlugin(
       BinaryMessenger* messenger);
 
-  virtual void NotifyWinEventWrapper(DWORD event,
-                                     HWND hwnd,
-                                     LONG idObject,
-                                     LONG idChild);
+  virtual void NotifyWinEventWrapper(ui::AXPlatformNodeWin* node, ax::mojom::Event event);
```

Comment:
`AXPlatformNodeWin` already has a helper method for dispatching accessibility events, so rather than dispatching them directly from the view, we use this helper method. This also means UIA events are handled alongside MSAA events.